sch

```diff
@@ -6,7 +6,7 @@ url=$(sort ~/.config/search/search | sed 's/:.*//' | dmenu -i -p "Search Engine"
 search=$(sort ~/.config/search/search_history | dmenu -i -p "Search")
 
 # Echo to history file
-if [ ! "$(grep -q "$search" < ~/.config/search/search_history)" ]; then
+if ! grep -q "$search" < ~/.config/search/search_history; then
     if [ "$(wc -l < ~/.config/search/search_history)" -gt 500 ]; then
```

Comment:
I know this isn't a change, but still:

Since `wc` is going to output only an integer, unless something bizarre occurs, I'd say the quotes here aren't needed, but that's probably just a me thing.

Also, I think that same file path is used a lot, so it might be a good candidate for a variable.